pcm_osx.cpp

```diff
@@ -5,9 +5,10 @@
 
 class PcmOutputMacOSX: public PcmOutput {
 public:
-    PcmOutputMacOSX();
+    PcmOutputMacOSX(int rate);
     virtual ~PcmOutputMacOSX();
-    virtual int getSampleRate() { return 44100; }
+    virtual int getSampleRate() { return sample_rate; }
+    virtual void setSampleRate(int rate) { sample_rate = rate; }
```

Comment:
There is.  All of the pcm_* classes support a setSampleRate(), which sets the class local int sample_rate.  When I made the changes to the global sample_rate variable, it will be much more clear.